ocaml/dune

```diff
@@ -51,7 +51,7 @@
  (modules snarky_js_types)
  (link_flags (-linkall))
  (modes native)
- (libraries mina_base fields_derivers_zkapps yojson)
```

Comment:
I would have done this in a different commit than the one udpating the artefacts.